src/Sentry/SentryBundle/Resources/config/services.yml

```
@@ -17,6 +17,7 @@ services:
           - '@?security.authorization_checker'
           - '@sentry.client'
           - '%sentry.skip_capture%'
+          - '@event_dispatcher'
```

Yes, the `@?` is used to silence failures for missing references and passing a null instead.